src/Cli/dotnet/commands/dotnet-workload/install/NetSdkManagedInstaller.cs

```diff
@@ -137,6 +140,16 @@ public void InstallWorkloadPack(PackInfo packInfo, SdkFeatureBand sdkFeatureBand
             }
         }
 
+        private PackageId GetPackageId(PackInfo packInfo)
+        {
+            var runtimeId = RuntimeInformation.RuntimeIdentifier;
+            var splitRuntimeId = RuntimeInformation.RuntimeIdentifier.Split('-');
+            var versionlessRuntimeId = $"{Regex.Replace(splitRuntimeId[0], @"[\d-]", string.Empty)}-{splitRuntimeId[1]}";
+            var packageId = packInfo.Aliases.ContainsKey(runtimeId) ? packInfo.Aliases[runtimeId].ToString() :
+                packInfo.Aliases.ContainsKey(versionlessRuntimeId) ? packInfo.Aliases[versionlessRuntimeId].ToString() : packInfo.Id;
```

Comment:
Is this the correct behavior here, defaulting to the pack id as the package id? Or should we error if aliases exist but there are no compatible runtime ids specified?

Reply:
This was clarified recently in https://github.com/dotnet/designs/pull/206 it should treat the package as not required on that platform.

Reply:
@mhutch